Add startrek Trivia list

scarecr0w12 opened the pull request:
Star trek trivia list

### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
Star trek trivia list

scarecr0w12 committed:
Add startrek Trivia list back from Old redbot

Star trek list from old redbot formatted for current YAML

scarecr0w12 committed:
Create 2946.enhance.rst

Vexed01 commented:
Please note that the bot is case sensitive with answers. As an example in [line 5](https://github.com/Cog-Creators/Red-DiscordBot/blob/47dc1caf61a164fcbe82cd291813bceeb884ee6a/redbot/cogs/trivia/data/lists/startrek.yaml#L5) you put the single answer as `pennsylvania`. This means that the bot will mark `Pennsylvania` incorrect. That question would ideally look like this:
```
In What State Is Carbon Creek Located?:
- pennsylvania
- Pennsylvania
```

scarecr0w12 commented:
Yeah i realized that after i got it set, im gonna go through the list again and add in alternative forms of the answers and capitalization. Wanted to get a first set up though.

Jackenmen committed:
Commit for triggering CI tests

Flame442 committed:
Merge branch 'V3/develop' into patch-1

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,3159 @@
+What Species Forwarded The Distress Signal To The Vulcan High Command?:
+- tellarite
+What Is A Vulcan Second Foremother?:
+- a Mother?s Mother?s Mother
```
Looks like a `'` got turned into a `?` here.
```suggestion
- a Mother's Mother's Mother
```

scarecr0w12 replied:
Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

aikaterna replied:
> Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

Hi there, I don't want this to sound like I'm pushing you but if you don't have the time for this I can get it wrapped up and approved, just let me know and I'll be happy to reformat and help. Please let us know soonish if you can, this item is slated to hopefully be included with the next update if everything goes well. Again sorry for being a bit pushy about time when this has been sitting for so long.

Flame442 reviewed (changes requested):
If you're still interested in working on this (sorry for the 3 year wait), the questions should have normal English capitalization rules for a sentence (only capitalize the first letter and proper nouns) and a lot of these questions could use alternative answers, especially the dates and names, to accept a bit more variety in how a user could input the answer. Note that trivia **is** case insensitive for answers, so you **do not** need to add different *casings*, only different *phrasings* (`august 19, 1921` could also be `8/19/1921` or `19/8/1921`).

Kowlin committed:
Update redbot/cogs/trivia/data/lists/startrek.yaml

Co-authored-by: Flame442 <34169552+Flame442@users.noreply.github.com>

Flame442 committed:
Merge branch 'V3/develop' into patch-1

aikaterna committed:
Reformat startrek.yaml

Formatted the entire file properly, removed redundant or incomplete questions, clarified many questions and answers, added a couple questions.

aikaterna committed:
Merge branch 'V3/develop' into patch-1

aikaterna commented:
This is ready for review, if there are any Star Trek nerds out there that would like to go over the changes to this file, besides my review request for Flame.

As a comment on this file for the future, the majority of the trivia questions are from Star Trek: The Original Series with a handful from The Next Generation, Voyager, Deep Space 9, and some of the older movies. If someone wanted to expand this list or add to it, I would suggest more entries from other parts of the Star Trek universe or spinoffs instead of the Original Series.

aikaterna committed:
Date typo

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,1975 @@
+- Tellarite
+What is a Vulcan second foremother?:
+- A mother's mother's mother
+What does Mestral use to free the trapped minors?:
```
```suggestion
What does Mestral use to free the trapped miners?:
```

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,1975 @@
+- Allasomorph
+In what episode was Captain Picard kidnapped by aliens and put into a room with other humans to study their behavior?:
+- Allegiance
+"In the Star Trek: The Original Series episode The Ultimate Computer, what planet did the M-5 computer scan to test it's performance with planetary contact?":
```
```suggestion
"In the Star Trek: The Original Series episode The Ultimate Computer, what planet did the M-5 computer scan to test its performance with planetary contact?":
```

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,1975 @@
+- Bitanium
+What is the most sacred planet in the Klingon empire, the supposed returning point of Kahless?:
+- Boreth
+What race uses a huge cube-shaped craft with a higly decentralized design?:
```
```suggestion
What race uses a huge cube-shaped craft with a highly decentralized design?:
```

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,1975 @@
+- 39
+How many Federation lives were lost at the Battle of Wolf 359?:
+- 11,000
+What was the name of the Borg that survivied the crash of his scout ship, who was captured by Enterprise-D?:
```
```suggestion
What was the name of the Borg that survived the crash of his scout ship, who was captured by Enterprise-D?:
```

Flame442 commented on redbot/cogs/trivia/data/lists/startrek.yaml:
```diff
@@ -0,0 +1,1975 @@
+- Julien
+The Next Generation proposal originally called for a female character in place of Wesley Crusher. What was her name?:
+- Leslie
+Which Next Generation episode was originally banned in Britain due to it's reference to terrorism and the IRA?:
```
```suggestion
Which Next Generation episode was originally banned in Britain due to its reference to terrorism and the IRA?:
```

Flame442 committed:
Apply suggestions from code review
